src/Azure.Functions.Cli/Kubernetes/KubernetesHelpers.cs

```diff
@@ -452,25 +451,6 @@ private static async Task<string> GetServiceIp(ServiceV1 service, int retryCount
             return string.Empty;
         }
 
-        private static IDictionary<string, string> GetFunctionKeys(IDictionary<string, string> currentImageFuncKeys, IDictionary<string, string> existingFuncKeys)
-        {
-            if ((currentImageFuncKeys == null || !currentImageFuncKeys.Any())
-                || (existingFuncKeys == null || !existingFuncKeys.Any()))
-            {
-                return currentImageFuncKeys;
-            }
-
-            //The function keys that doesn't exist in Kubernetes yet
-            IDictionary<string, string> funcKeys = currentImageFuncKeys.Except(existingFuncKeys, new KeyBasedDictionaryComparer()).ToDictionary(kvp => kvp.Key, kvp => kvp.Value);
```

Comment:
This line and line 465 seem to be taking the set difference between `currentImageFuncKeys` and `existingFuncKeys` and adding back the elements in their intersection, in essence doing nothing but creating a copy of `currentImageFuncKeys`.

Reply:
I don't think the names of the variables are super accurate because it seems like they have both keys _and_ values, and the values are where the difference comes in. So if the same key is defined in both `currentImageFuncKeys` and `existingFuncKeys` - I believe the value from `existingFuncKeys` is chosen over the value from `currentImageFuncKeys`.

Reply:
Right, thanks for the catch Eric. An interesting exception to this seems to be when `currentImageFuncKeys` is `null` but `existingFuncKeys` is not `null` or empty. In either case, I'll add this function back and make the changes to the `KeyBasedDictionaryComparer`.